CHANGELOG.md

```diff
@@ -6,7 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [?.??.?] - ????-??-??
+* #316 - BREAKING CHANGE addressing a typo - `http::server::Configuration::max_resp_handlers` renamed to `http::server::Configuration::max_resp_headers`
```

Comment:
Nit: Seems you merged/rebased wrongly? Most of what pops up as "new" here is really my earlier changes?

Reply:
yup, i somehow messed up with rebasing somewhere. dunno why git still considers this as an addition though.